docs/user-guide/kubectl-overview.md

```diff
@@ -135,6 +135,66 @@ For explanation, here I gave some mostly often used `kubectl` command examples.
 		// Starts streaming of logs from pod <pod-name>, it is something like 'tail -f'.
 		$ kubectl logs -f <pod-name>
 
+                // Get the logs from a previous run of a particular pod
+                $ kubectl logs --previous <pod-name>
+
+## Kubectl output
+
+`kubectl` supports a variety of output formats for different use cases.  The output format is selected with the `-o/--output` flag.
+
+The following options are supported:
+
+Output Format | Description
+--------------| -----------
+<default>     | Human readable format
```

Comment:
`<default>` is not rendered in github. Is this meant to state that if you dont specify -o=outputformat, then the default is still human readable format?

If thats the case, lets remove it from this table because it implies you need to specify -o=nothing.

Ill add mention of this default in the comment above, please delete this row from the table.